omnibus/config/software/datadog-agent-integrations.rb

```diff
@@ -71,6 +71,12 @@
         copy "#{check_dir}/conf.yaml.default", "#{check_conf_dir}/"
       end
 
+      # Copy the metric file, if it exists
+      if File.exist? "#{check_dir}/metrics.yaml"
+        mkdir check_conf_dir unless File.exists? (check_conf_dir)
```

Comment:
I think we should create the conf folder in advance and avoid checking if that exists at every step both here and while copying yaml files but this is going to change anyways and the code is readable, so let's keep it like it is